web/templates/user/_user_show_basedata.html

```diff
@@ -43,6 +43,7 @@
                 <span class="fa fa-home fa fa-home"></span>
                 <span>{{ user.address }}</span>
               {% endif %}
+              <a href="{{ url_for('.edit_user_address', user_id=user.id) }}"><span class="fa fa-edit"></span></a>
             {% elif user.address is not none %}
               <span class="fa fa-home"></span>
               <span>{{ user.address }}</span>
```

Comment:
The button for editing the users address should also be visible if no room is set.

Also: The else condition can be removed, as address cannot be `None`.

Reply:
Fixed